### ISSUE TITLE
Maintain Heroku Demo Site

### ISSUE THREAD

**Issue by katyedwards** [healthlocker/healthlocker#861]
Not sure if we mentioned this... 

Can we maintain a copy of HL on Heroku which we can continue to use demoing Evan and Robert etc but with no ePJS connection/ live data?

Presumably with a different URL?

**Comment by iteles** [comment]
@katyedwards Seems sensible! We briefly discussed this possibility a couple of days ago when you were over at dwyl but didn't specify it as a requirement. Good to have this in the backlog!

We'll discuss it next week and let you know the implications 👍

**Comment by nelsonic** [comment]
We _cannot_ connect to ePJS from Heroku for security reasons (SLaM's decision)
so once the ePJS link is live Heroku will no longer work.

**Comment by iteles** [comment]
@nelsonic I think the suggestion here is that we look to have a 'version' on heroku with made-up seed date for demos.

**Comment by katyedwards** [comment]
Yep, we just want a non-ePJS connected version for demo-ing, whether that is on heroku or staging site on azure - whatever is easier/ cheaper going forward.

**Comment by markwilliamfirth** [comment]
**Key things to update:**
Footer
Coping strategies alignment

**Rob to Research:**
- 10m Spike into creating clinicians / users on postgres side (Staging Site)
- Whether he is able to find most recent Heroku version of app to make quick updates (Heroku Site)

**Comment by katyedwards** [comment]
Waiting for new environment provision from SLaM - blocked!
